### ISSUE TITLE
Migrate TCP Business Days Script to New App

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#2298]
1. Create API Views
2. Update Knack script config

**Comment by dianamartin** [comment]
- [x] Created api view page and added `TCP tracking submissions` table

**Comment by amenity** [comment]
@johnclary @dianamartin - Sprint 27?

**Comment by johnclary** [comment]
Sprint 29 :). Marking as blocked by [Knackpy v1.0](https://github.com/cityofaustin/atd-data-tech/issues/3241)

**Comment by amenity** [comment]
Duplicate of #2317
